pkg/awsvpc/examples/input/input_aws_sg_1.json

```diff
@@ -1463,8 +1532,9 @@
             "IsDefault": true,
             "OwnerId": "OwnerId:8",
             "State": "available",
-            "Tags": [],
-            "VpcId": "VpcId:43"
+            "Tags": null,
+            "VpcId": "VpcId:40"
         }
-    ]
+    ],
+    "config-file-git-sha1": "$Id$"
```

Comment:
please notice,
the correct way to get a the config file from the teraform example repository is to it check out from the repository.